src/chocolatey/infrastructure.app/commands/ChocolateyUpgradeCommand.cs

```diff
@@ -63,6 +63,9 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                 .Add("params=|parameters=|pkgparameters=|packageparameters=|package-parameters=",
                      "PackageParameters - Parameters to pass to the package. Defaults to unspecified.",
                      option => configuration.PackageParameters = option.remove_surrounding_quotes())
+                .Add("applyToDependentPackages",

```

Comment:
This also applies to these as well:
 * https://github.com/chocolatey/choco/pull/915/files#diff-ecbbd2c62c5a004c0d087408a3c059e2R65
 * https://github.com/chocolatey/choco/pull/915/files#diff-d60cf2d264e4ef972e6ba3043c83c54bR61